### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jxing-brainstation**
- readme.md now has a summary of the lecture for gitflow
- also deleted outdated git branching commands file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-How to work with git:
+Let's learn about git flow!
 
-- never keep your develop branch up to date
-- merge your code into develop without a pull request
+1. keeping your develop branch up to date with the remote
+2. fixing merge conflicts
+3. creating and reviewing pull requests
```
Comment (jennyxing) [inline README.md:5]: Where did my changes go? 😈

**Review by jennyxing — CHANGES_REQUESTED** [review]
I also made some changes in the readme file but it look like your changes are overwriting mine. Is is what the team wanted?

**Comment by jxing-brainstation** [comment]
> I also made some changes in the readme file but it look like your changes are overwriting mine. Is is what the team wanted?

The team had a unanimous vote to keep my changes.

**Review by jennyxing — APPROVED** [review]
Okay this lgtm!

**Pull request merged**